app/lib/pre_assembly/digital_object.rb

```diff
@@ -40,7 +40,7 @@ def initialize(batch, stager:, container: '', stageable_items: nil, object_files
     # set this object's content_md_creation_style
     # @return [Symbol]
     def content_md_creation_style
-      # map the object type to content metadata creation styles supported by the assembly-objectfile gem
+      # map the object type to structural styles supported by the FileSetBuilder class
 
```

Comment:
adjusting this comment because i don't think we are using assembly-objectfile gem anymore for this purpose